src/modules/robot/Robot.cpp

```diff
@@ -396,22 +396,20 @@ void Robot::on_gcode_received(void *argument)
                     this->max_speeds[Y_AXIS] = gcode->get_value('Y');
                 if (gcode->has_letter('Z'))
                     this->max_speeds[Z_AXIS] = gcode->get_value('Z');
-                if (gcode->has_letter('A'))
-                    alpha_stepper_motor->set_max_rate(gcode->get_value('A'));
-                if (gcode->has_letter('B'))
-                    beta_stepper_motor->set_max_rate(gcode->get_value('B'));
-                if (gcode->has_letter('C'))
-                    gamma_stepper_motor->set_max_rate(gcode->get_value('C'));
-
+                for (size_t i = 0; i < actuators.size(); i++) {
+                    if (gcode->has_letter('A' + i))
```

Comment:
Another option for setting many steppers or positions would be letter pair codes. Like M203 PA12 PB33 SA400 RD1500, where the first letter of the pair would be "P"  for position, "S" actuator, or "R" for raw step count. The second letter would be the channel: "A","B","C",... We could make the second character a digit but that make it hard to read. Perhaps this wants to be a separate pull request.